### DISCUSSION TITLE
[PROPOSAL] Fairness in seasonal harvesting quests

### DISCUSSION THREAD

**Discussion by brian-man** [sunflower-land/sunflower-land#2334] [General]
### Overview

The intent for adding the seasonal tickets (as I understand it) to minting limited supply collectibles is to improving access and fairness -- i.e. preventing all collectibles from being gobbled up only by whales.

### Problem

Some of the quests/chores from the seasonal NPC are accomplished with harvesting.  Harvesting time is sensitive to duration boosts.

### Examples

- Mysterious Parsnip
- Carrot Amulet
- Nancy/Scarecrow/Kuebiko
- Lunar Calendar
- Angel / Devil Wings

### Proposal

- Option A

Scale tickets rewarded downward for harvest quests according to the growth time.

For example, if a crop with a base growth time of an hour is boosted to 1/2 hour then the ticket reward should be halved for completing that quest.

- Option B

Scale required crops upward for harvest quests according to the growth time.

For example, if a crop with a base growth time of an hour is boosted to 1/2 hour then the required crops for the quest should be doubled.

Note that for Wings this option is problematic as the required crops would be, effectively, infinity.  So an alternative formulation would be needed.  Maybe use 4x?

**Comment by TheDcol** [comment]
I think the intent of seasonal tickets was to introduce a way for players to obtain items more via gameplay versus 'gas luck', which frustrated many people before. I THINK..

Old item release: 2000 people trying to mint a 200-500 supply nft clicking at the same second resulting in errors and pain.

So.. i think PART of the intent was to help with this issue, to make it 'fairer' in that way.

No doubt players with time boosts are at an advantage with seasonal tickets, but i do not think that should change as that might be the intended feature for them... 

Sidenote: I think those with better farms (5 gold nodes, 5 iron, 40+ plots, 10+ trees and stones) have an even BIGGER advantage than time boosts, especially if there are more 5 gold/5 iron/10 stone requirements. I have 5+ gold nodes and jumped ahead of people by quite a lot in the race. This to me feels wrong as players do not have a choice with the lands they have, but they have some choice in having minted or bought items that have time boosts.

The ideas you proposed are sound, but not needed, in my opinion. Land discrepancy is a bigger issue and one i am looking forward to be addressed soon

**Reply by brian-man** [reply]
> The ideas you proposed are sound, but not needed, in my opinion. Land discrepancy is a bigger issue and one i am looking forward to be addressed soon

That's already being addressed separatelly, hence not bringing it up here.

Once that's addressed, the fundamental brokenness remaining is for wings.  The other growth affecting boosts are arguably similarly problematic (just at a lesser scale), so a common solution seems appropriate.

**Reply by blastaman** [reply]
Both are a significant handicap. I have 39 plots, 9 stone, 4 iron and 3 gold and managed to overtake a lot of better farms during the race and mint a cabbage girl easy… because I have mysterious parsnip, app beaver, legacy skills etc. Mind you I also use fertilisers on the kale quests. But again that gave an advantage other players could obtain.
Now that resource balance is coming, I feel like I could be first to mint rarest item next season instead of the 250th.

**Comment by blastaman** [comment]
Was thinking about how the race could be more fair in the future. Another option could be to disable time related boosts from sfts  for the quests. For example if the quest is grow 100 cabbage and you have a cabbage girl then once you accept the quest your cabbages take the normal amount of time to grow until the quest is completed. Problem is where to draw the line?
I would add Luna Calendar and Goblin Scarecrow to the list of advantage giving sfts. And then it has been pointed out the those with crop Legacy skills also get a further advantage with another 10% reduction on growth time for crops.

Anyway the intent of the seasons is obviously not clear. Maybe the intent is for whales to scoop up all the collectables… this line in the docs makes me think so:
“ A new player would likely go for a high supply decoration, whereas an OG player would go for the high-end items.”

**Reply by blastaman** [reply]
Just to add to my comment…. There is no reason we can’t have different rules for different seasons. If the intent is for ‘OG players’ to have a significant advantage during the seasons I would love to see at least one season that gives all players that a grind a fair chance.

**Comment by ghost** [comment]
Explore my ideas about the quest system, I think they are more interesting in terms of gameplay https://github.com/sunflower-land/sunflower-land/discussions/2326
